src/pages/Rooms/_HeaderActions.js

```diff
@@ -50,6 +91,14 @@ const mapDispatchToProps = (dispatch, { room: slug }) => ({
     }
 
     return dispatch(deleteSubscription(slug, onSuccess, onError))
+  },
+
+  handleArchive: () => {
+    return dispatch(archiveRoom(slug))
+  },
+
+  handleReactivate: () => {
+    return dispatch(reactivateRoom(slug))
   }
```

Comment:
Although it's good practice to add a linebreak between function definitions, when inside a object the rule of not adding a linebreak between key/values should win. Let's drop those extra lines.